_maps/RandomRuins/LavaRuins/lavaland_surface_seed_vault.dmm

```diff
@@ -1,708 +1,69 @@
-/obj/item/weapon/reagent_containers/glass/bucket,
-/turf/open/floor/plasteel/freezer,
-/area/ruin/powered/seedvault)
-"J" = (
-/obj/item/weapon/storage/toolbox/syndicate,
-/obj/structure/table/wood,
-/turf/open/floor/plasteel/freezer,
-/area/ruin/powered/seedvault)
-"L" = (
-/obj/structure/disposalpipe/segment{
-	dir = 1;
-	icon_state = "pipe-c"
-	},
-/turf/open/floor/plating/asteroid/basalt/lava_land_surface,
-/area/lavaland/surface/outdoors)
-"M" = (
-/obj/structure/disposalpipe/trunk{
-	dir = 8
-	},
-/turf/open/floor/plating/asteroid/basalt/lava_land_surface,
-/area/lavaland/surface/outdoors)
-"N" = (
-/obj/structure/table/wood,
-/obj/item/weapon/storage/box/disks_plantgene,
-/obj/machinery/light{
-	dir = 1
-	},
-/turf/open/floor/plasteel/freezer,
-/area/ruin/powered/seedvault)
-"O" = (
-/obj/machinery/light{
-	dir = 1
-	},
-/turf/open/floor/plasteel/freezer,
-/area/ruin/powered/seedvault)
-"P" = (
-/obj/machinery/hydroponics/constructable,
-/obj/machinery/light{
-	dir = 1
-	},
-/turf/open/floor/plasteel/freezer,
-/area/ruin/powered/seedvault)
-"Q" = (
-/turf/closed/wall/r_wall,
-/area/ruin/powered/seedvault)
-"R" = (
-/obj/machinery/light{
-	dir = 8
-	},
-/turf/open/floor/plasteel/freezer,
-/area/ruin/powered/seedvault)
-"S" = (
-/obj/machinery/light{
-	dir = 4
-	},
-/turf/open/floor/plasteel/freezer,
-/area/ruin/powered/seedvault)
-"U" = (
-/obj/machinery/light,
-/turf/open/floor/plasteel/freezer,
-/area/ruin/powered/seedvault)
-"V" = (
-/obj/structure/disposalpipe/segment,
-/turf/closed/wall/r_wall,
-/area/ruin/powered/seedvault)
-"X" = (
-/obj/machinery/hydroponics/constructable,
-/obj/machinery/light,
-/turf/open/floor/plasteel/freezer,
-/area/ruin/powered/seedvault)
-"Z" = (
-/obj/effect/baseturf_helper/lava_land/surface,
-/turf/closed/wall/r_wall,
-/area/ruin/powered/seedvault)
+"a" = (/turf/closed/mineral/volcanic/lava_land_surface,/area/lavaland/surface/outdoors)
+"b" = (/turf/open/floor/plating/asteroid/basalt/lava_land_surface,/area/lavaland/surface/outdoors)
+"c" = (/turf/closed/wall/r_wall,/area/ruin/powered/seedvault)
+"d" = (/obj/structure/table/wood,/obj/item/weapon/lighter,/obj/item/weapon/lighter,/obj/item/weapon/storage/fancy/rollingpapers,/obj/item/weapon/storage/fancy/rollingpapers,/obj/item/weapon/storage/fancy/rollingpapers,/obj/item/weapon/storage/fancy/rollingpapers,/turf/open/floor/plasteel/freezer,/area/ruin/powered/seedvault)
+"e" = (/obj/structure/table/wood,/obj/item/weapon/storage/box/disks_plantgene,/obj/machinery/light{dir = 1},/turf/open/floor/plasteel/freezer,/area/ruin/powered/seedvault)
+"f" = (/obj/structure/table/wood,/obj/item/weapon/storage/box/disks_plantgene,/turf/open/floor/plasteel/freezer,/area/ruin/powered/seedvault)
+"g" = (/obj/machinery/plantgenes/seedvault,/turf/open/floor/plasteel/freezer,/area/ruin/powered/seedvault)
+"h" = (/obj/structure/table/wood,/obj/item/weapon/disk/design_disk/strange_seed{step_x = 0; step_y = 1},/turf/open/floor/plasteel/freezer,/area/ruin/powered/seedvault)
```

Comment:
i did that purely so i could (hopefully) get issues id been having with the map merger working again if you read the small print, im on my way to rectifying it.